crates/bevy_gltf/src/loader.rs

```diff
@@ -817,6 +831,9 @@ fn load_material(
 
         let metallic_roughness_texture = pbr.metallic_roughness_texture().map(|info| {
             // TODO: handle info.tex_coord() (the *set* index for the right texcoords)
+            if info.texture_transform().map(texture_transform_mat3) != Some(uv_transform) {
+                warn!("Only the texture transform on the base color texture is supported, ignoring the texture transform on the metallic/roughness texture");
```

Comment:
Having the texture label in those warnings would make it easier to identify which texture raised the warning for the user.

Reply:
[`7298cdb` (#11904)](https://github.com/bevyengine/bevy/pull/11904/commits/7298cdbe62891229c8e99edff358a51757ccc941)